requirements-docs.txt

```diff
@@ -1,9 +1,10 @@
-jupyter_core<=4.8.2
-docutils<=0.17.1
+jupyter_core
+docutils
 sphinx
 sphinx_rtd_theme
 nbsphinx
 matplotlib
 ipython
 m2r2
-jinja2==3.0.3
+jinja2
+pytest
```

Comment:
Since `pytest` is on `autodoc_mock_imports` list it probably doesn't have to be here.

Reply:
i actually believe that `pytest` needs to be included in the regular requirements, since we use it in `testing_utils.py`

Reply:
No, we can always say that `sentinelhub.testing_utils` is a part of `DEV` package extension. I would definitely not add it to the main requirements which are already bloated.

Reply:
works without it, removed